airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -305,8 +305,10 @@ def __init__(
         self.delegate_to = delegate_to
         self.impersonation_chain = impersonation_chain
 
-        if not objects and not prefix:
-            raise ValueError("Either object or prefix should be set. Both are None")
+        if objects is None and prefix is None:
```

Comment:
what about replacing this line with the below snippet?
```
if (objects or prefix) is None:
    
```

Reply:
I think the current way is easier and more explicit.